commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentMethodGroupRelsDisplayContext.java

```diff
@@ -151,9 +151,11 @@ else if (navigation.equals("inactive")) {
 			results = addDefaultCommercePaymentMethodGroupRels(results);
 		}
 
-		results.sort(
-			new CommercePaymentMethodGroupRelNameComparator(
-				themeDisplay.getLocale()));
+		if (!results.isEmpty()) {
+			results.sort(
```

Comment:
Hey @andrea-ale-sbarra, let's have a call about this tomorrow. I think this isn't the right fix.